### DISCUSSION TITLE
Is there a way to make line thickness uniform for line charts?

### DISCUSSION THREAD

**Discussion by treyholt** [ChartsCSS/charts.css#82] [Q&A]
I'm noticing that when there is a stark difference in height - width ratio between table rows, it results in a varied thickness of lines along the line chart. See the picture below for an example of what I mean (notice the first line is thin and tall, and the others are thicker). 
Curious if this is a known issue, or if there's a known way to manipulate the css to account for this. Will look into it myself if not. 

<img width="257" alt="Screen Shot 2022-09-13 at 2 38 59 PM" src="https://user-images.githubusercontent.com/35981934/189983587-77607225-6976-40e0-93ce-fd45df70d842.png">

**Comment by ramiy** [comment]
It's a knows issue. If you have ideas how to fix that, please share.

**Comment by jem-n-em** [comment]
hello, if this issue has not had a PR submitted for it, would you mind me taking a look into this issue?

**Comment by jem-n-em** [comment]
@ramiy or @treyholt do any of you have an idea of where to begin looking in the source code?  I've been toying around with lines in the Line Chart section of dist/charts.css and inside src/charts/_line.scss and I am not sure what I can change to toy with these lines.

**Comment by ramiy** [comment]
Hi @jem-n-em

As a user, you can change the line thickness using the `--line-size` custom property.

**Comment by russelporosky** [comment]
@jem-n-em The `clip-path` rules in [_line.scss](https://github.com/ChartsCSS/charts.css/blob/main/src/charts/_line.scss) are what draw the lines I believe.
